lib/vtls/wolfssl.c

```diff
@@ -499,6 +499,13 @@ wolfssl_connect_step1(struct connectdata *conn,
   }
 #endif /* OPENSSL_EXTRA */
 
+#ifdef HAVE_SECURE_RENEGOTIATION
```

Comment:
I take it this is the define you set "manually" then to enable this in the build?

Reply:
This define is part of wolfssl, which get generated when you configure wolfssl with feature (--enable-secure-renegotiation). Command example:

./configure  --enable-postauth --enable-tls13 --enable-libwebsockets --enable-harden --enable-aesni --enable-psk --enable-stacklog --enable-sni --enable-alpn --enable-trustedca --enable-truncatedhmac --enable-stunnel --enable-certservice --enable-enckeys --enable-secure-renegotiation --enable-debug


Above command generate wolfssl/options.h, which has this macro defined, if feature was enable while configuring the woflss. Now if you look at the line https://github.com/curl/curl/blob/b150a63d5cee71a4f318ac613823588bd7b24256/lib/vtls/wolfssl.c#L35, we include this header in this file compilation.

Reply:
Aha, thanks!